homework.md

```diff
@@ -44,37 +44,38 @@ Using CSS, how would I select every `<h1>` tag on the page?
 
 What is JavaScript?
 
-A programming language used on the internet
+JavaScript is a text-based programming language used both on the client-side and server-side that allows you to make web pages interactive.
 
 ## Question 8
 
 What three problem does Git & Github solve?
```

Comment:
Hi Khalid, try to write full sentences when adding the three problems. Answers sounds fine.

Reply:
Thanks I will update